orchagent/mirrororch.cpp

```diff
@@ -816,6 +816,12 @@ bool MirrorOrch::updateSessionType(const string& name, MirrorEntry& session)
 {
     SWSS_LOG_ENTER();
 
+    deactivateSession(name, session);
+    activateSession(name, session);
+
+    return true;
+
+#if 0
```

Comment:
if we do not want to code, I suggest we delete it, and also do we have vstest for this case?